doctrine/mongodb-odm-bundle/3.3/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "bundles": {
+        "Doctrine\\Bundle\\MongoDBBundle\\DoctrineMongoDBBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "etc/": "%ETC_DIR%/",
+        "src/": "%SRC_DIR%/"
+    },
+    "env": {
+        "MONGODB_URL": "mongodb://localhost:27017",
```

Comment:
Can you suffix the database name to keep consistency with Doctrine ORM recipe and then simply have
`"MONGODB_URL": "mongodb://localhost:27017/symfony",` ?